yt_dlp/extractor/daftsex.py

```diff
@@ -0,0 +1,87 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_b64decode
+from ..utils import (
+    int_or_none,
+    js_to_json,
+    get_elements_by_class,
+    parse_duration,
+    try_get
+)
+
+
+class DAFTSEXIE(InfoExtractor):
```

Comment:
Only the first character of each word must be upper case.
```suggestion
class DaftsexIE(InfoExtractor):
```